tests/playwright.config.js

```diff
@@ -16,15 +16,18 @@ const ADMIN_STORAGE_STATE = path.join(__dirname, '/.auth/admin_storage_state.jso
 /**
  * @see https://playwright.dev/docs/test-configuration
  */
+const HEADFUL = !!process.env.SHOW_BROWSER;
```

Comment:
Is SHOW_BROWSER a custom property you added to your .env file? If so, does changing this property need a restart of the e2e container? If it does, it might be easier to just change the value of `headless` to `true` or `false` in this file, since that won't require a restart.